install.sh

```diff
@@ -13,21 +13,20 @@ fail() {
 }
 
 usage() {
-  log "$0 [--head]"
+  log "$0 [--head][--version]"
```

Comment:
Well that should be just print list of commands that are possible. I don't really know how to show that in command lines.